### ISSUE TITLE
Error Importing Library

### ISSUE THREAD

**Issue by d4rkj0y** [rmcrackan/Libation#688]
**Describe the bug**
A clear and concise description of what the bug is. When launching the app, the auto scan doesn't download or add new book from my audible account. From both US and UK domains

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to the 'App menu bar'
2. Click on 'Import'
3. Scroll down to 'Scan Library of ALL accounts.'
4. See error - Scan indicates commencing "Scanning 2 accounts". No obvious app activity for 32 seconds (timed) then an "Error Importing Library" pop-up with instructions/ options appears.
5. To add: I also deleted my 2 accounts, restarted the app, and ran the wizard to re-add my 2 accounts but the error still occurred after this attempt to resolve.

**Expected behaviour**
Expected App to  Auto update library from launch (as per settings) with newly purchased or included in membership audible books. 

**Screenshots**
Screenshot with pop up
<img width="1673" alt="Screenshot 2023-07-28 at 15 12 57" src="https://github.com/rmcrackan/Libation/assets/110789808/6f7e89dd-a53f-493b-a914-bb2e449588c4">

Screenshot with no error pop-up and library updated 
<img width="1673" alt="Screenshot 2023-07-28 at 15 24 27" src="https://github.com/rmcrackan/Libation/assets/110789808/fa8fe689-db22-46c1-907f-e339bde792fc">

**Platform**
[e.g. Windows 10, Windows 11, Mac, Linux (State distribution)]
MacBook Pro (Retina, 15-inch, Mid 2014) Big Sur 11.7.9

**Log Files**
Attach your Libation log file here.
[Log202307.zip](https://github.com/rmcrackan/Libation/files/12195376/Log202307.zip)

In Conclusion:

As I was logging this 'bug' I still had the app running in the background not minimised. I noticed some activity from the visible part of the app window behind my browser while I was typing and it had updated my library. Which is good!! So to clarify the app was indeed working correctly I scanned All accounts again. This time no pop-up appeared. But I'm still reporting this because an error occurred and you may still find information about why it happened.
Thank you....It's a great program by the way and your efforts on it are much appreciated

**Comment by Mbucari** [comment]
Thanks for taking the time to report! I'll try to look into it this weekend.

**Comment by Mbucari** [comment]
@d4rkj0y This looks like it was a short-lived networking error. They're rare, but when they do happen they're usually short-lived.

Glad everything is working again!
